roles/pulp_repos/defaults/main.yml

```diff
@@ -0,0 +1,15 @@
+---
+
+# defaults file for pulp_repos
+pulp_rhel_codeready_repo:
```

Comment:
We are actually lacking documentation for this in:
https://github.com/pulp/pulp_installer/blob/main/roles/pulp_common/README.md#role-variables

Please document it along the lines of the rhel7_optional_repo .